test/object-deep-omit/index.cjs

```diff
@@ -0,0 +1,43 @@
+var test = require('../util/test')(__filename);
+var deepOmit = require('../../../packages/object-deep-omit/index.cjs');
```

Comment:
Looks like these paths are off (should be `'../../packages/object-deep-omit/index.cjs'` ?